src/_tokens/tokens.ts

```diff
@@ -77,7 +77,7 @@ export const color = {
 };
 
 export const fontFamily = {
-  sans: '"Nunito Sans", "Helvetica Neue", Helvetica, Arial, sans-serif',
+  sans: 'var(--nunito-sans, "Nunito Sans"), "Helvetica Neue", Helvetica, Arial, sans-serif',
```

Comment:
@JonathanKolnik Would this change break other site properties?

Reply:
It shouldn't because the syntax is that the original font family "Nunito Sans" is the fallback for when the variable isn't defined